release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -404,6 +404,50 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
 [id="ocp-4-16-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
```

Comment:
This sentence can be removed. It would then be consistent with surrounding components.  Although, technically correct to add a lead-in sentence, release notes are that big different. Each note mentions what is happening in this release.